airgun/entities/hostcollection.py

```diff
@@ -172,6 +176,39 @@ def install_errata(self, entity_name, errata_id,
             )
             return job_status_view.overview.read()
 
+    def manage_module_streams(self, entity_name, action_type, module_name,
+                              stream_version, customize=False, customize_values=None):
+        """ Manage module streams
+        :param str entity_name:  The host collection name.
+        :param action_type: remote action to execute on content host. Action value can be one of
+            them e.g. 'Enable', 'Disable', 'Install', 'Update', 'Remove', 'Reset'
+        :param str module_name:  The name of module on which action is performed
+        :param str stream_version:  The version of module on which action is performed
+        :param customize: special action type which should call on content host module streams
+            tab to run some custom expression related to module streams
+        :param customize_values: need to pass if run some custom command expression in content host
+
+        :returns job status view values
+
+        """
+        if customize_values is None:
+            customize_values = {}
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.details.manage_module_streams.click()
+        view = HostCollectionManageModuleStreamsView(view.browser)
+        view.search('name = {} and stream = {}'.format(module_name, stream_version))
+        if customize:
+            action_type = dict(is_customize=customize, action=action_type)
```

Comment:
In `airgun/entities/contenthost.py`, this is done unconditionally. Please make these two files consistent in that regard.